templates/index.html

```diff
@@ -50,8 +49,90 @@ <h1>Very fast server provisioning for your data centre</h1>
   </div>
 </section>
 
-<section class="p-strip">
+<section class="p-strip u-fixed-width">
```

Comment:
We shouldn't mix `u-fixed-width` or `row` with the strip. Can you add an inner `row`?

Reply:
I did that, sorry. But is it causing any issues? Seems to work ok and requires less markup.

Reply:
The issue with this is that it will confuse the next developer that comes to modify this code as it does not adhere to the coding standards from Vanilla. 

This will have an unexpected effect on common strip patterns such as `is-bordered` or changing the type of this strip to `p-strip--light`. Therefore, this breaks the common language between design and development.

A major concern when writing any form of code is to try and make it as clear and easy for the next person that reads it to understand and modify.

Reply:
Makes sense, I've changed it to the accepted form.